corehq/apps/domain/utils.py

```diff
@@ -100,7 +100,7 @@ def guess_domain_language(domain_name):
 
 
 @task(serializer='pickle', queue='background_queue')
-def send_repeater_payloads(repeater_id, payload_ids, email_id):
+def send_repeater_payloads(repeater_id, payload_ids, email_id, owner_id):
```

Comment:
I ended up just killing this feature 
https://github.com/dimagi/commcare-hq/pull/28273

Reply:
🎊